test/ng/animateCssSpec.js

```diff
@@ -126,7 +126,7 @@ describe("$animateCss", function() {
       var cancelSpy = jasmine.createSpy();
       var runner = $animateCss(element, { addClass: 'cool' }).start();
 
-      runner.catch(cancelSpy);
+      runner['catch'](cancelSpy);
```

Comment:
Since we try to support Android 2.3 in 1.5, I think it's fine to try to ensure tests at least parse there. ;P